internal/ethereum/blocklistener.go

```diff
@@ -185,6 +231,8 @@ func (bl *blockListener) listenLoop() {
 				log.L(bl.ctx).Errorf("Failed to establish new block filter: %s", err.Message)
 				failCount++
 				continue
+			} else {
```

Comment:
nit: There is a continue above so no else needed

Reply:
good catch, thanks